extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -15,6 +15,67 @@ const BackbeatConsumer = require('../../../lib/BackbeatConsumer');
 const BackbeatClient = require('../../../lib/clients/BackbeatClient');
 const QueueEntry = require('../utils/QueueEntry');
 
+class _UserAuthManager {
+    constructor(authConfig, log) {
+        assert.strictEqual(authConfig.type, 'user');
+
+        this.log = log;
+        const userInfo = authdata.users.find(
+            user => user.name === authConfig.user);
+        if (userInfo === undefined) {
+            throw Error(`No such user registered: ${authConfig.user}`);
```

Comment:
This error is about a misconfiguration: the friendly "user" name configured in config.json to reference a source or target account configured in authdata.json is not found in the latter, which should not happen if the product is configured correctly, so I think it's appropriate to throw here.